diff --git a/tests/selenium/wdio-mediawiki/wdio-defaults.conf.js b/tests/selenium/wdio-mediawiki/wdio-defaults.conf.js
index 3fee5fa014e..40714f45e3f 100644
--- a/tests/selenium/wdio-mediawiki/wdio-defaults.conf.js
+++ b/tests/selenium/wdio-mediawiki/wdio-defaults.conf.js
@@ -20,7 +20,7 @@ import { logBrowserInformation, logSystemInformation, makeFilenameDate, saveScre
 
 let xvfbProcesses = [];
 
-if ( !process.env.MW_SERVER || !process.env.MW_SCRIPT_PATH ) {
+if ( !process.env.MW_SERVER || process.env.MW_SCRIPT_PATH === undefined ) {
 	throw new Error( 'MW_SERVER or MW_SCRIPT_PATH not defined.\nSee https://www.mediawiki.org/wiki/Selenium/How-to/Set_environment_variables\n' );
 }
 
